lib/ical/binary.js

```diff
@@ -155,7 +155,24 @@ ICAL.Binary = (function() {
 
       dec = tmp_arr.join('');
 
-      return dec;
+
+      // expand for codes > 128 
+      tmp_arr = []; cntrl = 0;
+      i =0, ac=0; 
+      do {
+        h1 = dec.charCodeAt(i++)
+        h2 = dec.charCodeAt(i)
```

Comment:
I think the i and i++ lines here may be a bit confusing. I'd suggest using a for loop and then indexing with `i` and `i+1`.